### PR TITLE
CrossRef Harvester & Normalizer

### PR DESCRIPTION
**Pull request by caseyrollins**

### CODE REVIEW

File: providers/org/crossref/normalizer.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    title = ctx.title[0]
+    description = ctx.maybe('subtitle')
+    doi = ctx.DOI
+    contributors = ctx.author['*']
+    tags = ctx.maybe('subject')['*']
+
+
+class Contributor(Parser):
+    person = ctx
+    order_cited = ctx['index']
```
Comment (chrisseto) [inline providers/org/crossref/normalizer.py]: Leave a note about that

File: providers/org/crossref/harvester.py [hunk 2]
```diff
@@ -1,5 +1,32 @@
-from share.harvest.harvester import Harvester
+from furl import furl
+
+from share import Harvester
 
 
 class CrossRefHarvester(Harvester):
-    pass
+    url = 'http://api.crossref.org/v1/works'
```
Comment (chrisseto) [inline providers/org/crossref/harvester.py]: https

**Comment by caseyrollins** [comment]
@chrisseto 🐯  (haven't fully run this through the system yet though)

File: providers/org/crossref/normalizer.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+from share.normalize import *
+
+
+class CreativeWork(Parser):
+    # Dates in CrossRef metadata are often incomplete, see: https://github.com/CrossRef/rest-api-doc/blob/master/rest_api.md#notes-on-dates
+    title = ctx.title[0]
+    description = ctx.maybe('subtitle')
+    doi = ctx.DOI
```
Comment (caseyrollins) [inline providers/org/crossref/normalizer.py]: Prepend ```settings.DOI_BASE_URL``` to make this an actual URL.

File: providers/org/crossref/normalizer.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    contributors = ctx.author['*']
+    tags = ctx.maybe('subject')['*']
+
+
+class Contributor(Parser):
+    # CrossRef does not return a complete cited name, but returns the given name and family name of the contributor
+    person = ctx
+    order_cited = ctx['index']
```
Comment (chrisseto) [inline providers/org/crossref/normalizer.py]: We'll have to make this work but
```python
cited_name = ctx.given + ctx.given
# Or
cited_name = ctx.given + ' ' + ctx.given
# Or
cited_name = Join(ctx.given, ctx.given, using=' ')
```

**Comment by caseyrollins** [comment]
@chrisseto 🐘

**Pull request merged**